### ISSUE TITLE
Text using brand color on white background fails contrast check

### ISSUE THREAD

**Issue by markphd** [newwwie/newwwie.com#108]
As [pointed out by Cordelia](https://newwwie.slack.com/archives/CF54BSVT5/p1709516657102449?thread_ts=1708873005.829649&cid=CF54BSVT5), testing out the main brand color on white text fails accessibility. 

<img width="242" alt="Screenshot 2024-03-04 at 10 47 56 PM" src="https://github.com/newwwie/newwwie.com/assets/1754675/e4305391-24dd-4d52-89df-66d639d82e32">

Test checker result: https://colourcontrast.cc/85d4ed/ffffff

**Comment by Calinator444** [comment]
Working on this

**Comment by markphd** [comment]
Looks good to me, ok to close
